java/runtime/src/main/java/org/ray/runtime/util/generator/RayCallGenerator.java

```diff
@@ -8,18 +8,18 @@
 import org.apache.commons.io.FileUtils;
 
 /**
- * A util class that generates `RayCall.java`, which provides type-safe interfaces for `Ray.call`
- * and `Ray.createActor`.
+ * A util class that generates `RayCall.java` and `ActorCall.java`, which provide type-safe
+ * interfaces for `Ray.call`, `Ray.createActor` and `actor.call`.
  */
 public class RayCallGenerator extends BaseGenerator {
 
   /**
    * @return Whole file content of `RayCall.java`.
    */
-  private String build() {
+  private String generateRayCallDotJava() {
```

Comment:
Maybe `generateRayCallJava`?

Reply:
I thought about that. But that seemed more vague. So I ended up with this name. But, anyway, I think the documentation is clear enough.